hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/ExecutionContext.java

```diff
@@ -311,6 +318,13 @@ public void provideDynamicMetrics(MetricDescriptor descriptor, MetricsCollection
         }
         descriptor = descriptor.withTag(MetricTags.JOB, idToString(jobId))
                                .withTag(MetricTags.EXECUTION, idToString(executionId));
+
+        long executionStartTime = startTime.get();
+        if (executionStartTime != 0) {
+            context.collect(descriptor, EXECUTION_START_TIME, ProbeLevel.INFO, ProbeUnit.MS, executionStartTime);
+            context.collect(descriptor, EXECUTION_DURATION, ProbeLevel.INFO, ProbeUnit.MS,
```

Comment:
let's remove this, as it's just a derived value and obsolete as soon as it's collected